Makefile

```diff
@@ -0,0 +1,13 @@
+all: get build
+
+get:
```

Comment:
I think this entire section can be replaced by `go get -t ./...`

Reply:
I am a go newbie and can't seem to figure out the build system flags.